packages/react-ui-core/src/Counter/Counter.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+export default class Counter extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      count: this.props.count,
+    }
+    this.handleClick = this.handleClick.bind(this)
+  }
+  handleClick(count) {
+    this.props.handleClick(count)
+    this.setState({
+      count,
+    })
+  }
+  render() {
+    const { count } = this.state
+    const { theme, label, decrementUnit, incrementUnit } = this.props
+    return (
+      <div className={theme.Counter}>
+        <div className={theme['Counter-label']}>
+          {label}
+        </div>
+        <div className={theme['Counter-label-active']}>
+          <span
+            onClick={() => this.handleClick(this.state.count + 1)}
+            className={classNames('incrementUnit', theme['Counter-content'], theme['Counter-shape'])}
+          >
+            {this.props.decrementOperator}
+          </span>
+          <span
+            className={classNames(theme['Counter-content'], theme['Counter-count-content'], theme['Counter-Shape'])}
+          >{`${decrementUnit} ${count} ${incrementUnit}`}</span>
+          <span
+            onClick={() => this.handleClick(this.state.count - 1)}
```

Comment:
What if I want to decrement / increment by something like say 2? How do I do that?